pkg/cmd/bootstrap.go

```diff
@@ -40,6 +40,8 @@ var supportedDrivers = drivers{
 	"gitlab",
 }
 
+var sealedSecretDefaultValue = types.NamespacedName{Namespace: sealedSecretsNS, Name: sealedSecretsController}
```

Comment:
This is the default sealed secrets service name, right?  Rename it to something like defaultSealedSecretsServiceName?   Also, should we put it togetter with the var above?  
```
var (
    supportedDrivers = drivers{	"github", "gitlab" }
    ...
)

Reply:
Sure William will do!